test/clojure/core_test/portability.cljc

```diff
@@ -14,6 +14,7 @@
   ;; In CLJS, all numbers are really doubles and integer? and int?
   ;; return true if the fractional part of the double is zero
   #?(:cljs (integer? n)
+     :jank (cpp/jank.runtime.is_big_integer n)
      :default
      (and (integer? n)
           (not (int? n)))))
```

Comment:
If we're going to switch to this predicate, I think we could do a better job than this. Personally I think the default should be something like

```clj
(= (type 0N) (type n))
```